### DISCUSSION TITLE
[Question][Hw5] Unsupported instructions

### DISCUSSION THREAD

**Discussion by nv-quan** [prosyslab-classroom/cs348-information-security#606] [Q&A]
Hello,
I am a little bit confused by the following sentence: "You can ignore the other LLVM instructions marked as raise Unsupported". Does it mean that I can mark instructions other than those define in [this document](https://github.com/prosyslab-classroom/cs448-2021-spring-hw5-nv-quan/blob/master/doc/homework.pdf) as Unsupported when dealing with them in semantics.ml?
Thank you.

**Comment by KihongHeo** [comment]
Sorry. That is my mistake. Ignore that sentence and do not raise an exception by yourself.

**Comment by nv-quan** [comment]
Does it mean that the input program will only have the instruction type defined in the document? For example if the instruction is a binary operation, it can only be one of `add`, `sub`, `mul`, `sdiv`, and not  `or`, `xor`, `srem`, etc.?

**Comment by KihongHeo** [comment]
Yes. See Section 6 in README. Ignore all bitwise operators and rem.

**Comment by nv-quan** [comment]
Thank you.
